package.json

```diff
@@ -1,88 +1,93 @@
 {
-	"name": "jquery-migrate",
-	"title": "jQuery Migrate",
-	"description": "Migrate older jQuery code to jQuery 3.0+",
-	"main": "dist/jquery-migrate.js",
-	"version": "3.4.2-pre",
-	"homepage": "https://github.com/jquery/jquery-migrate",
-	"author": {
-		"name": "OpenJS Foundation and other contributors",
-		"url": "https://openjsf.org"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git://github.com/jquery/jquery-migrate.git"
-	},
-	"bugs": {
-		"url": "http://bugs.jquery.com/"
-	},
-	"license": "MIT",
-	"scripts": {
-		"build": "grunt default-no-test",
-		"test": "grunt test",
-		"ci": "grunt ci"
-	},
-	"peerDependencies": {
-		"jquery": ">=3 <4"
-	},
-	"devDependencies": {
-		"chalk": "5.0.1",
-		"commitplease": "3.2.0",
-		"enquirer": "2.3.6",
-		"eslint": "8.11.0",
-		"eslint-config-jquery": "3.0.0",
-		"eslint-plugin-import": "2.25.4",
-		"grunt": "1.5.3",
-		"grunt-cli": "1.4.3",
-		"grunt-compare-size": "0.4.2",
-		"grunt-contrib-uglify": "4.0.1",
-		"grunt-contrib-watch": "1.1.0",
-		"grunt-eslint": "24.0.0",
-		"grunt-karma": "4.0.2",
-		"grunt-npmcopy": "0.2.0",
-		"gzip-js": "0.3.2",
-		"karma": "6.3.17",
-		"karma-browserstack-launcher": "1.6.0",
-		"karma-chrome-launcher": "3.1.1",
-		"karma-firefox-launcher": "2.1.2",
-		"karma-qunit": "4.1.2",
-		"load-grunt-tasks": "5.1.0",
-		"native-promise-only": "0.8.1",
-		"qunit": "2.19.1",
-		"rollup": "2.70.1",
-		"testswarm": "1.1.2"
-	},
-	"keywords": [
-		"jquery",
-		"javascript",
-		"browser",
-		"plugin",
-		"migrate"
-	],
-	"commitplease": {
-		"components": [
-			"Docs",
-			"Tests",
-			"Build",
-			"Release",
-			"Core",
-			"Ajax",
-			"Attributes",
-			"Callbacks",
-			"CSS",
-			"Data",
-			"Deferred",
-			"Deprecated",
-			"Dimensions",
-			"Effects",
-			"Event",
-			"Manipulation",
-			"Offset",
-			"Queue",
-			"Selector",
-			"Serialize",
-			"Traversing",
-			"Wrap"
-		]
-	}
+  "name": "jquery-migrate",
+  "title": "jQuery Migrate",
+  "description": "Migrate older jQuery code to jQuery 3.0+",
+  "main": "dist/jquery-migrate.js",
+  "version": "3.4.2-pre",
+  "homepage": "https://github.com/jquery/jquery-migrate",
+  "author": {
+    "name": "OpenJS Foundation and other contributors",
+    "url": "https://openjsf.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jquery/jquery-migrate.git"
+  },
+  "bugs": {
+    "url": "http://bugs.jquery.com/"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "grunt",
+    "test:browser": "npm run build && npm run test:unit -- -b chrome -b firefox -h",
+    "test:safari": "npm run build && npm run test:unit -- -v -b safari",
+    "test:server": "node test/runner/server.js",
+    "test:unit": "node test/runner/command.js"
```

Comment:
Can you also add a `test` script that does everything that's sensible to do locally? It's customary to expect `npm install && npm test` - or its short form `npm it` to install dependencies, run the build & all tests that should verify your change didn't break anything. We also have this in Core.

`npm test` should not run Safari tests so that it can be run on Windows.

Reply:
I think it will just be an alias for `test:browser` right now, until we migrate all grunt tasks.

Reply:
I'm fine with that. Still, it's good to have the `test` script defined.

I think you could also add linting to that, though.

Reply:
`test:browser` includes `npm run build`, which runs `grunt`, which includes `grunt lint`.